File: app/src/main/java/dk/itu/moapd/sensorsmonitor/MainActivity.java
package dk.itu.moapd.sensorsmonitor;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;

import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN);

        setContentView(R.layout.activity_main);

        FragmentManager fragmentManager =
                getSupportFragmentManager();
        Fragment fragment = fragmentManager
                .findFragmentById(R.id.fragment);

        if (fragment == null)
            fragmentManager.beginTransaction()
                    .add(R.id.fragment, new MainFragment())
                    .commit();
    }

}

File: app/src/main/java/dk/itu/moapd/sensorsmonitor/MainActivity.kt
package dk.itu.moapd.sensorsmonitor

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Window
import android.view.WindowManager

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        requestWindowFeature(Window.FEATURE_NO_TITLE)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN)

        setContentView(R.layout.activity_main)

        val fragment = supportFragmentManager
            .findFragmentById(R.id.fragment)

        if (fragment == null)
            supportFragmentManager
                .beginTransaction()
                .add(R.id.fragment, MainFragment())
                .commit()
    }

}
